src/js/components/AuthorPicker.js

```
@@ -2,261 +2,129 @@
-        const newSelectedIds = [...selectedAuthorIds, parseInt(authorId)];
-        onChange(newSelectedIds);
-        setAuthorId(''); // Reset selection
-    };
-
-    // Handle author removal
-    const handleRemoveAuthor = (id) => {
-        const newSelectedIds = selectedAuthorIds.filter(authorId => authorId !== id);
-        onChange(newSelectedIds);
-    };
+    // Prepare options for select
+    const authorOptions = [
```

ESLint

[error] Replace `····` with `↹`

(prettier/prettier)